README.md

```diff
@@ -77,3 +85,84 @@ For homebrew, see the following: https://gist.github.com/mxcl/1173223
 If you take these steps, obviously also remove your ROS workspace and start the install
 process over from scratch as well. Finally, audit your `$PATH` variable to ensure that
 when you run `python`, you're getting Homebrew's `python`.
+Another way to check which Python you are running is to do:
+
+```bash
+which python # Should result in /usr/local/bin/python
+ls -l $(which python) # Should show a symlink pointing to Homebrew's Cellar
+```
+
+If you are getting permission errors when you `sudo uninstall` pip packages,
+see [Issue #11](https://github.com/mikepurvis/ros-install-osx/issues/11) and
+[this StackOverflow Q&A](http://stackoverflow.com/a/35051066/2653356).
+
+### El Capitan support
+
+The `install` script may not work as smoothly in OS X El Capitan.
+Here are some pointers, tips, and hacks to help you complete the installation.
+This list was compiled based on the discussion in [Issue #12](https://github.com/mikepurvis/ros-install-osx/issues/12).
+
+#### No definition of [procps] for OS [osx]
+
+This issue is known to the developers and is being addressed. See:
+
+* http://answers.ros.org/question/224956/no-definition-of-procps-for-os-osx
+* https://github.com/ros-perception/vision_opencv/pull/109
+
+#### FLTK fails to build
+
+For a quick fix, you can do the following:
+
+```bash
+brew install fltk --devel
```

Comment:
Sounds good Mike.

And yeah, it's still a problem. It happened to me on Thursday 01/28.